README.md

```diff
@@ -181,9 +181,12 @@ _We test only sources written by us and we don`t need to test external sources(o
 You can find it in [Coverage report](https://codecov.io/gh/smartdevicelink/sdl_core/branch/develop)
 
 ### How to get Test Coverage locally
- 1. Build project with enabled flag _-DBUILD_TESTS=on_
+ Precondition : if you dont have lcov, gcov, html2text, install it before running.
+ 1. Build project with flags _-DBUILD_TESTS=on -DENABLE_GCOV=on_
  2. Execute command 'make test'
- 3. Execute './tools/Utils/collect_coverage.sh <path_to_build_directory>'
+ 3. Execute './tools/infrastructure/collect_coverage.sh \<path_to_build_directory\>'
+
+ Note : In case of lots of warnings execute : _sudo ln -sf /usr/bin/gcov-4.9 /usr/bin/gcov_
```

Comment:
@anosach-luxoft, If you don't mind - I will answer. As we use gcc version 4.9 instead of default gcc version provided by OS - we need to create symlink from gcov 4.9 to system gcov used by collect_coverage script. 
@LuxoftAKutsan Please add necessary information.